lms/urls.py

```diff
@@ -224,6 +224,7 @@
     url(r'^support/', include('lms.djangoapps.support.urls')),
 ]
 
+# Todo -- check its useability
```

Comment:
A little more description with all caps todo.

Reply:
removing that in my next commit. thanks for raising it 👍

Reply:
Reminder to remove this `TODO`.